webserver/views/datasets.py

```diff
@@ -308,7 +308,11 @@ def edit(dataset_id):
 def delete(dataset_id):
     ds = get_dataset(dataset_id)
     if ds["author"] != current_user.id:
-        raise Unauthorized("You can't delete this dataset.")
+        return jsonify(
```

Comment:
Why was this changed from the exception to a custom error message? Is it because you needed the `error=True` value in the returned data?